logo/licensing.md

```diff
@@ -0,0 +1,14 @@
+---
+title: Licensing of the beanvalidation logo
+author: Hendrik Ebbers
+---
+
+Copyright 2017 Hendrik Ebbers (http://www.guigarage.com/about)
+
+Redistribution and use in source and binary forms, with or without modification, are permitted provided that the following conditions are met:
```

Comment:
It would be nice to have the name of the license. It's easier to get compatibility information and know what you can do or not when you know which license it is.